sdk/freeport/freeport.go

```diff
@@ -381,11 +394,39 @@ func logf(severity string, format string, a ...interface{}) {
 	fmt.Fprintf(os.Stderr, "["+severity+"] freeport: "+format+"\n", a...)
 }
 
+type TestingT interface {
```

Comment:
`sdk/freeport` is a library, so did you want this to be exported and thus part of `sdk/freeport` forever?

Reply:
I think so. This follows the Go convention of "accept interface, return structs". 

Instead of accepting a large interface I think it's common to define a minimal interface so that if someone needs to use this library and doesn't have a `*testing.T` they can easily make a fake implementation.

I think I should probably document this interface better. It should say that new methods could be added to the interface, but generally those methods should always be implemented by `*testing.T`.